vendor/mixins.py

```diff
@@ -1,15 +0,0 @@
-# TODO: Need to think of a better way to check if a user has product in a mixin
```

Comment:
What replaced this mixin?  Keep in mind that this is an Open Source project and removing things without warning is dangerous.

Reply:
That file was replaced with vendor/views/mixin.py with the mixin we actually use: `ProductRequiredMixin` I had created that file and View as a starting point to approach how should we check if a user owns a product. Decided to remove the file as it probably is not even used.

Should I add a warning in the notes then?